### DISCUSSION TITLE
Migrations Postgres migrate on wrong table

### DISCUSSION THREAD

**Discussion by Alitindrawan24** [goravel/goravel#213] [Q&A]
I created a migration and ran the migration, but what happened was that the migration created a table in the postgres database registered in DB_USERNAME instead of my_app registered in DB_DATABASE.

Here is my configuration in .env

`
DB_CONNECTION=postgresql
`
`
DB_HOST=localhost
`
`
DB_PORT=5432
`
`
DB_DATABASE=my_app
`
`
DB_USERNAME=postgres
`
`
DB_PASSWORD=
`

Thank you

**Comment by hwbrzzl** [comment]
Migrate to the `postgres` database? Sorry, I can't reproduce it locally. Can you please provider more information?

**Reply by Alitindrawan24** [reply]
Yes I run the step like this :
- change `DB_CONNECTION` in .env file into postgresql
- create migration using `go run . make:migration` command
- run migrate using `go run . make:migrate` command
- the result is table migrated into postgres table not my_app table

Here is my .env file configuration
```
DB_CONNECTION=postgresql
DB_HOST=localhost
DB_PORT=5432
DB_DATABASE=my_app
DB_USERNAME=postgres
DB_PASSWORD=
```

is something wrong that I did ? Thank you

**Reply by hwbrzzl** [reply]
Please send a screenshot of the DB tool, let's check it first.

**Reply by Alitindrawan24** [reply]
Sure, here my screenshot for DB tools and migrations configuration

.env configuration
<img width="233" alt="Screenshot 2023-07-20 at 10 00 43" src="https://github.com/goravel/goravel/assets/36658186/fabd8d22-6010-4d67-bd4f-9ffa32cbdfb1">

Table my_app after migration
<img width="1332" alt="Screenshot 2023-07-20 at 10 00 35" src="https://github.com/goravel/goravel/assets/36658186/93580ad8-4fe2-4ac1-8785-68c0dc4e5e6b">

Table postgres after migration
<img width="1333" alt="Screenshot 2023-07-20 at 10 00 15" src="https://github.com/goravel/goravel/assets/36658186/8d67cc57-ab4d-4fb4-b7fd-b50379a812f3">

**Reply by hwbrzzl** [reply]
Could you please show your migration files code here?

**Reply by Alitindrawan24** [reply]
sure, here `20230712171846_create_short_links_table.up`

```
CREATE TABLE short_links (
  id serial PRIMARY KEY NOT NULL,
  short_link varchar(100) NOT NULL,
  actual_link text NOT NULL,
  created_at timestamp NOT NULL,
  updated_at timestamp NOT NULL
);
```

**Reply by hwbrzzl** [reply]
Do you change `config/database.go`? Please post the file content(screenshot) here.

**Reply by Alitindrawan24** [reply]
Here is my `config/database.go`

<img width="923" alt="Screenshot 2023-07-22 at 17 20 34" src="https://github.com/goravel/goravel/assets/36658186/e91cb3e7-0663-4103-8997-dc64ce04411c">

**Reply by hwbrzzl** [reply]
Emm, interesting, it seems to be no problem, I can't reproduce your issue locally, could you please create a PR for github.com/goravel/example so that I can reproduce it? Or what's your version of Goravel, you can upgrade to the latest version and try again if it isn't.
